src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -73,9 +73,7 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
                 this._orderActionCreator.submitOrder({
                     ...payload,
                     payment: paymentPayload,
-                    // Note: API currently doesn't support using Store Credit with Klarna.
-                    // To prevent deducting customer's store credit, set it as false.
-                    useStoreCredit: false,
+                    useStoreCredit: true,
```

Comment:
I think the value should come from `payload.useStoreCredit` ?

Reply:
yes you are right, thanks for the review